types/selenium-webdriver/test/bidi/logInspector.ts

```diff
@@ -0,0 +1,15 @@
+import * as webdriver from "selenium-webdriver";
+import LogInspector from "selenium-webdriver/bidi/logInspector";
+
+function TestLogInspector() {
+    const driver: webdriver.WebDriver = new webdriver.Builder().build();
+    const logInspector = LogInspector(driver);
+    let lastLog;
```

Comment:
```suggestion
    let lastLog: GenericLogEntry | undefined;
```
Gotta use the types so the type assertions work